src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -99,6 +112,8 @@ public class PullRequestHandler : IHttpHandler
             {
                 case "master":
                 case "origin/master":
+                case "origin/release/1.0.0-rc2":
+                case "origin/infrastructure":
```

Comment:
* We have branches release/1.0.0-rc1 and StressTests which will cause the PR service to be nuked from the test server box. These should not be accessible by anyone and should return 400. 
* CI will test release/1.0.0-rc2 branch periodically and we need that to return 200 (the alternative is to churn the netci.groovy file, which is harder to deal with, but will still not guard against someone hitting the URI directly). We can get rid of this once we stop periodically testing the rc2 branch. 
* CI will test infrastructure branch periodically - we want that to work until we merge. 

Unfortunately, the way this service is deployed, we cannot avoid either a white or blacklist; one of these will exist.